File: JAVASCRIPT/Dinner2/leap_year.java
public class Nested{
public static void main(String[]args){

count = 0;
count1 = 0;
number = 1900;
while (number <= 2025){
	number ++;
	if (number % 4 == 0 && number % 100 != 0){
		count +=1;
	}
	if (number % 4 == 0 && number % 100 == 0 && number % 400 == 0){
		count1 +=1;
	}
System.out.print(count+count1);

}
}
}

File: JAVASCRIPT/Dinner2/leap_year.py
for number in range(1900, 2026, 1):
	if number % 4 == 0 and number % 100 != 0:
		print(number)
	if number % 4 == 0 and number % 100 == 0 and number % 400 == 0:
		print(number)